src/templates/dandelion/config/screens/LockScreen.js

```diff
@@ -0,0 +1,338 @@
+import React, { useCallback, useReducer, useState, useRef } from 'react'
+import PropTypes from 'prop-types'
+import { Field, GU, Help, Info, TextInput, isAddress } from '@aragon/ui'
+import {
+  Header,
+  Navigation,
+  ScreenPropsType,
+  Duration,
+  KnownAppBadge,
+} from '../../../kit'
+import TokenSelector from '../../components/TokenSelector/TokenSelector'
+import { getDefaultLockTokenByNetwork } from '../helpers/tokens'
+import { shortenAddress } from '../../../../web3-utils'
+
+const MINUTE_IN_SECONDS = 60
+const HOUR_IN_SECONDS = MINUTE_IN_SECONDS * 60
+const DAY_IN_SECONDS = HOUR_IN_SECONDS * 24
+const SPAM_PENALTY_DEFAULT = 50
+
+const DEFAULT_DURATION = DAY_IN_SECONDS
+
+const DURATION_LABEL = 'lock duration'
+const DURATION_HELP_TEXT = `Lock Duration, The lock duration is the period of time that the tokens will be locked. `
+
+function useFieldsLayout() {
+  // In its own hook to be adapted for smaller views
+  return `
+    display: grid;
+    grid-template-columns: auto ${12 * GU}px;
+    grid-column-gap: ${1.5 * GU}px;
+  `
+}
+
+function validationError(duration, tokenAddress, lockAmount) {
+  if (!tokenAddress || !isAddress(tokenAddress)) {
+    return 'You need at least one valid address.'
+  }
+  if (!lockAmount > 0) {
```

Comment:
```suggestion
  if (lockAmount <= 0) {
```

This previously did not do what you were expecting it to do :).